src/libsync/excludedfiles.h

```diff
@@ -70,7 +67,6 @@ public slots:
     // but the pointer can be in a csync_context so that it can itself also query the list.
     c_strlist_t** _excludesPtr;
     QStringList _excludeFiles;
-    mutable QReadWriteLock _mutex;
```

Comment:
See a4e0899af4eec5883bb16011c5760ca0c3707f76, it's always called from the GUI thread.